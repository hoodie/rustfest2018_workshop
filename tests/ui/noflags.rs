fn flagged_fn(flag: bool) {

}

fn main() {

}