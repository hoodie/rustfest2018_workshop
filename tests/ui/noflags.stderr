warning: don't use boolean flags
 --> $DIR/noflags.rs:1:1
  |
1 | / fn flagged_fn(flag: bool) {
2 | |
3 | | }
  | |_^
  |
  = note: #[warn(boolargs)] on by default

